dev/AppLifecycle/AppInstance.cpp

```diff
@@ -33,30 +34,37 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         // Push past the '----' commandline argument prefix.
         argsStart += 4;
 
-        // We explicitly use find_first_of here, so that the resulting data may contain : as a valid character.
-        auto argsEnd = commandLine.find_first_of(L":", argsStart);
+        auto argsEnd = commandLine.find_first_of(L" ", argsStart);
```

Comment:
`find_first_of` with a string pointer is wasteful if you're searching for only one character. Use `find_first_of(L' ', argsStart)` instead.